### DISCUSSION TITLE
Number of default examples not reset on web UI although set load_examples=false

### DISCUSSION THREAD

**Discussion by tailutuan** [apache/airflow#35452] [General] [answered]
I followed below instructions:
- Deploy airflow docker compose: [docker-compose.yaml](https://airflow.apache.org/docs/apache-airflow/2.7.2/docker-compose.yaml)
- Tried `load_examples=false` or `AIRFLOW__CORE__LOAD_EXAMPLES: 'false' `

But on UI webserver that number of default examples not reset
![image](https://github.com/apache/airflow/assets/148857802/79e0b45b-b0e9-4e99-b49e-7bdbc5d7f0f2)

Is this bug or author wanna show this?

**Comment by tailutuan** [comment]
Sorry @potiuk because bothered you :smile:

**Reply by potiuk** [reply]
Don't be sorry. Just don't do it

**Reply by LengYue12389** [reply]
This requires deleting the information of the example DAGs in the metabase database

**Comment by Taragolis** [comment, marked as answer]
Hey, I guess this is a bug with counters in Airflow 2.7.2

- https://github.com/apache/airflow/pull/34944
- https://github.com/apache/airflow/issues/34955

Which should be resolved in 2.7.3, it would be nice if you could test it on RC1:
- https://github.com/apache/airflow/issues/35372
